node_modules/oae-tests/lib/util.js

```diff
@@ -983,8 +983,8 @@ var createInitialTestConfig = module.exports.createInitialTestConfig = function(
 
     // log everything (except mocha output) to tests.log
     config.log.streams = [{
-        'level': 'info',
-        'path': './tests.log'
+        'level': config.test.level,
```

Comment:
Could you set these to `config.test.level || 'info'` and `config.test.path || `'./tests.log'`? The default values should be here instead of in the env file, so that they work in environments other than local, too.